archive/Gcl_Archive/Associate-Cloud-Engineer/Set-Up-an-App-Dev-Environment-on-Google-Cloud.html

```
@@ -0,0 +1,1056 @@
+            <h2><span class="station__chip"></span>コース全体像とラーニングパス</h2>
+          </div>
+        </div>
+        <p>
+          このコースは「写真管理アプリ Memories」というシナリオを軸に、<strong>ストレージ・権限管理・サーバーレス処理・非同期通知</strong>という、モダンなアプリ開発環境の基本4要素を一気通貫で学びます。
+        </p>
+
+        <div class="diagram">
+          <p class="diagram__cap">Fig 2.1 — 学習パイプラインの全体像</p>
+          <pre class="mermaid">
+flowchart LR
+    A["① Cloud Storage<br/>バケット作成・オブジェクト操作"] --> B["② Cloud IAM<br/>権限の確認・付与・削除"]
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)